Rocket.Chat/NewRoomViewController.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  NewRoomViewController.swift
+//  Rocket.Chat
+//
+//  Created by Bruno Macabeus Aquino on 27/09/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class NewRoomViewController: BaseViewController {
```

Comment:
Again, since we don't want to introduce conflicting nomenclature, can you use CreateChannel... instead of NewRoom...?